### DISCUSSION TITLE
NLP using SNN

### DISCUSSION THREAD

**Discussion by palla-lin** [jeshraghian/snntorch#180] [Q&A]
Hi,
I am newer. I am wondering how to convert language into a spike train. Could someone give me some guidelines? I only see some examples for CV. Thanks

**Comment by ridgerchu** [comment]
Hello,
To my knowledge, SNN is mainly utilized in CV field. However, there exist some new works implement SNN into some well-known language models, such as [TextCNN](https://openreview.net/forum?id=pgU3k7QXuz0). This work supplements its code for reproducing, which employs snntorch for construct the model. You can find the implementation by this link(https://openreview.net/attachment?id=pgU3k7QXuz0&name=supplementary_material).
